integrations/actix/src/lib.rs

```diff
@@ -98,6 +98,10 @@ impl ResponseOptions {
 /// Provides an easy way to redirect the user from within a server function. Mimicking the Remix `redirect()`,
 /// it sets a [StatusCode] of 302 and a [LOCATION](header::LOCATION) header with the provided value.
 /// If looking to redirect from the client, `leptos_router::use_navigate()` should be used instead.
+#[cfg_attr(
+    any(debug_assertions, feature = "ssr"),
```

Comment:
I think for all of these in the Actix integration it can be simplified to just
```rust
#[instrument(level = "trace", skip_all)]
```
because it's always SSR.

Reply:
True, I got lazy and just copy pasted things at one point